openhtf/core/measurements.py

```diff
@@ -139,6 +140,19 @@ def _initialize_value(self):
     else:
       self.measured_value = MeasuredValue(self.name)
 
+  def __deepcopy__(self, memo):
```

Comment:
Another option would be to make `__init__` do this, right? It would be simpler in there, and useful for other non-copying code as well